### DISCUSSION TITLE
Auto create schema from table

### DISCUSSION THREAD

**Discussion by sbscan** [pocketbase/pocketbase#2582] [Q&A] [answered]
I'm imported my old tables into sqlite but can't see the fields in pocketbase because I need to edit _collections table and edit schema for the collection. Is there or any plans to sync _collections table with the actual tables and create the schema automaticly?

**Comment by ganigeorgiev** [comment, marked as answer]
There is no sync option at the moment. There are plans for tables "autodiscovery" in the future, but for now remains a low priority.

**Reply by sbscan** [reply]
I understand that you focus on speed and stability but for many people who want to migrate existing data will ask this feature.
I'll do with JS for now, but maybe you should think about the priority of this. The more people bring their data, the more popular.
BTW thanks for this great project, every elite developer is talking about it.
